src/braket/circuits/gates.py

```diff
@@ -1954,21 +1979,27 @@ def bind_values(self, **kwargs) -> GPi:
     @staticmethod
     @circuit.subroutine(register=True)
     def gpi(
-        target: QubitInput, angle: Union[FreeParameterExpression, float]
+        target: QubitSetInput,
```

Comment:
Why not `QubitInput` if this is a 1q gate?

Reply:
Our single-qubit gates are 50/50 on whether they broadcast over multiple qubits when provided with more than one. If you look at the first few gates, you'll find they all do. According to the implementation, so does GPi. Some gates differ from this behavior and I think we should make it consistent; not convinced it has to happen this PR though

Reply:
Oh, good catch!